clients/go/admin/client.go

```diff
@@ -144,6 +150,18 @@ func InitializeAdminClient(ctx context.Context, cfg Config) service.AdminService
 	return NewAdminClient(ctx, adminConnection)
 }
 
+// Create an AdminClient and AuthServiceClient with a shared Admin connection for the process
+func InitializeClients(ctx context.Context, cfg Config) (service.AdminServiceClient, service.AuthServiceClient) {
```

Comment:
I think we might want to create and expose a wrapper similar to how kube api client behaves... so you get back one client that has .Admin() and .Auth()

Reply:
Done